sdk/identity/identity/tests.yml

```diff
@@ -33,3 +33,4 @@ stages:
         IDENTITY_SP_CERT_SNI: $(Agent.TempDirectory)/testsni.pfx
         IDENTITY_SP_CERT_SNI_PEM: $(Agent.TempDirectory)/testsni.pem
         IDENTITY_PEM_CONTENTS: $(net-identity-spcert-pem)
+        AZURE_IDENTITY_BROWSER_CLIENT_ID: $(identity-azure-sdk-js-test-browser-client-id)
```

Comment:
We need to create an app registration with an “spa” redirect endpoint and store it in the keyvault used for the CI en variables 🙂